### DISCUSSION TITLE
Two different output configurations for the same provider type?

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#991] [Configuration] [answered]
I tried to have to different collections in the service file referring to the same "type" in the provider file, but a got an error when I did that. The point was to have two different output configurations in the two collections based on the same provider type name. Is it possible?

**Comment by cportele** [comment]
Yes, that is possible. How does the configuration look like and what is the error that you get?

**Reply by Sverre-Iversen** [reply]
**I think I did it right this time? - no errors**
This is my Service bunnfellingsomrader.yml:
<pre>
collections:
  bunnfellingflate: # simple configuration - complex structure output
    id: bunnfellingflate
...
  bunnfellingflate0: # extended configuration - flat structure  output
    id: bunnfellingflate0
...
    - buildingBlock: FEATURES_CORE
      enabled: true
      featureProvider: bunnfellingsomrader # <- new
      featureType: bunnfellingflate # <- new: collection bunnfellingflate & bunnfellingflate0 
      # <- points to the same provider, but have different formatted output (not shown here)
...
</pre>
**This is the right way?**

**Reply by cportele** [reply, marked as answer]
Yes, this looks correct (note that `featureProvider` should not be necessary as it is the default value).
